### PR TITLE
Feature/deploy

### PR DESCRIPTION
**Pull request by xal9wiii4ik**

### CODE REVIEW

File: .github/workflows/deploy_ecr.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ dev ]
```
Comment (artsiom-streetshares) [inline .github/workflows/deploy_ecr.yml]: the idea is that you build and push the image on successful merge to your main branch. this is achieved by `push`. why you need `pull_request`? with this condition it's gonna be treggered on pull request open and close https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

File: .github/workflows/deploy_ecr.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+name: Deploy to ECR
```
Comment (artsiom-streetshares) [inline .github/workflows/deploy_ecr.yml]:
```suggestion
name: Build and push to ECR
```

**Review by artsiom-streetshares — COMMENTED** [review]
rename workflow to `push_ecr`

File: .github/workflows/deploy_ecr.yml (hunk 2 above)
Comment (xal9wiii4ik) [reply .github/workflows/deploy_ecr.yml]: done

File: .github/workflows/deploy_ecr.yml (hunk 1 above)
Comment (xal9wiii4ik) [reply .github/workflows/deploy_ecr.yml]: done

**Review by artsiom-streetshares — APPROVED** [review]

**Pull request merged**